package.json

```diff
@@ -32,6 +32,7 @@
     "@types/yargs": "^15.0.11",
     "@typescript-eslint/parser": "^5.31.0",
     "chai": "^4.1.2",
+    "devtools-protocol": "^0.0.1040073",
```

Comment:
required to fix type-check, was failing because the puppeteer-core that lighthouse depends on isn't enough to satisfy our internal type usage (we have a devDep that uses a newer devtools-protocol...)